src/main/java/com/afornalik/ox/model/player/PlayerContainer.java

```diff
@@ -0,0 +1,35 @@
+package com.afornalik.ox.model.player;
+
+import com.afornalik.ox.model.board.FieldStatus;
+
+import java.util.Map;
+
+public class PlayerContainer {
```

Comment:
Is this class even necessary?

Reply:
This class hold all players attending the game. I will add some functionality to this class e.g validation.